src/Servers/Kestrel/test/BindTests/AddressRegistrationTests.cs

```diff
@@ -98,6 +98,7 @@ public async Task RegisterIPEndPoint_IPv6StaticPort_Success()
         }
 
         [ConditionalTheory]
+        [MinimumOSVersion(OperatingSystems.Windows, WindowsVersions.Win81)]
```

Comment:
Is this because of the port 0 binding? Add a comment on this line about that.

@halter73 is that really limited to Win8?

Reply:
Specifically, the test fails for binding to https urls on win7. Again, I'm just empirically adding this to everything that failed during validation.

Reply:
What's the error? Https should work on Win7, unless this is because we're using new certs?

Reply:
https://helix.dot.net/api/2019-06-17/jobs/d7959616-94c9-48af-bcd0-6407db053881/workitems/Sockets.BindTests--net5.0/console
```
[xUnit.net 00:00:03.03]     RegisterIPEndPoint_DynamicPort_Success(endPoint: [::1]:0, testUrl: "https://[::1]") [FAIL]
[xUnit.net 00:00:03.03]       System.Security.Authentication.AuthenticationException : Authentication failed, see inner exception.
[xUnit.net 00:00:03.03]       ---- System.ComponentModel.Win32Exception : The client and server cannot communicate, because they do not possess a common algorithm.
[xUnit.net 00:00:03.03]       Stack Trace:
[xUnit.net 00:00:03.03]            at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](TIOAdapter adapter, Boolean receiveFirst, Byte[] reAuthenticationData, Boolean isApm)
[xUnit.net 00:00:03.03]         /_/src/Testing/src/HttpClientSlim.cs(155,0): at Microsoft.AspNetCore.Testing.HttpClientSlim.GetStream(Uri requestUri, Boolean validateCertificate)
[xUnit.net 00:00:03.03]         /_/src/Testing/src/HttpClientSlim.cs(30,0): at Microsoft.AspNetCore.Testing.HttpClientSlim.<>c__DisplayClass1_0.<<GetStringAsync>b__0>d.MoveNext()
[xUnit.net 00:00:03.03]         --- End of stack trace from previous location ---
[xUnit.net 00:00:03.03]         /_/src/Testing/src/HttpClientSlim.cs(116,0): at Microsoft.AspNetCore.Testing.HttpClientSlim.RetryRequest(Func`1 retryBlock)
[xUnit.net 00:00:03.03]         /_/src/Testing/src/HttpClientSlim.cs(28,0): at Microsoft.AspNetCore.Testing.HttpClientSlim.GetStringAsync(Uri requestUri, Boolean validateCertificate)
[xUnit.net 00:00:03.03]         /_/src/Testing/src/HttpClientSlim.cs(24,0): at Microsoft.AspNetCore.Testing.HttpClientSlim.GetStringAsync(String requestUri, Boolean validateCertificate)
[xUnit.net 00:00:03.03]         /_/src/Servers/Kestrel/test/BindTests/AddressRegistrationTests.cs(319,0): at Microsoft.AspNetCore.Server.Kestrel.FunctionalTests.AddressRegistrationTests.RegisterIPEndPoint_Success(IPEndPoint endPoint, String testUrl, Int32 testPort)
[xUnit.net 00:00:03.03]         /_/src/Servers/Kestrel/test/BindTests/AddressRegistrationTests.cs(105,0): at Microsoft.AspNetCore.Server.Kestrel.FunctionalTests.AddressRegistrationTests.RegisterIPEndPoint_DynamicPort_Success(IPEndPoint endPoint, String testUrl)
[xUnit.net 00:00:03.03]         --- End of stack trace from previous location ---
[xUnit.net 00:00:03.03]         ----- Inner Stack Trace -----
[xUnit.net 00:00:03.03]         
```

Note, I didn't investigate why the test is failing and I'm just turning it off to fix the build.

Reply:
Don't just turn tests off without filing a tracking issue.
Is this rebased on @halter73's change to the default kestrel TLS protocol version?

Reply:
Ah I wasn't clear. I meant the issue https://github.com/dotnet/aspnetcore/issues/22917 to track the investigation after I disable the tests.

Reply:
That should be linked in the attribute.

Reply:
If you mean https://github.com/dotnet/aspnetcore/pull/22437, these tests were run after the change was merged.